homeassistant/components/tuya/icons.json

```
@@ -156,6 +165,9 @@
       },
       "target_humidity": {
         "default": "mdi:water-percent"
+      },
+      "sensor_choose": {
```

`sensor_choose` => should be `sensor_choice` but maybe `sensor_selection` is better?
```suggestion
      "sensor_selection": {
```